ApollosProject/apollos-prototype
# Corrects the coordinates being sent to LocationFinder query

## DESCRIPTION

### What does this PR do, or why is it needed?

Currently, the query in the locationfinder is passing the wrong coordinates for our campusCard list. It's sending the `initialRegion` which is set to be somewhere in Kansas(I think?) for the US map. This PR changes the coordinates to the User's coordinates so that the distance to location is correct.

### What design trade-offs/decisions were made?

I've set the default coordinates as the differential office in Cincinnati. This can be anything though.

### How do I test this PR?

Navigate to the Location in the user-settings and make sure the distance to location is correct. Or go through onboarding and do the same.

### What automated tests did you write?

N/A.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
